main.go

```diff
@@ -36,10 +36,16 @@ var (
 	clusterCmd = app.Command("cluster", "Do some action on cluster(s).")
 
 	domainCmd     = app.Command("domain", "Do some action on domain(s).")
-	domainCluster = domainCmd.Flag("cluster", "Cluster ID.").Short('c').String()
+	domainCluster = domainCmd.Flag("cluster", "Cluster ID. This flag requires a domain ID, domain name will not work.").Short('c').String()
 
 	projectCmd     = app.Command("project", "Do some action on project(s).")
-	projectCluster = projectCmd.Flag("cluster", "Cluster ID.").Short('c').String()
+	projectCluster = projectCmd.Flag("cluster", "Cluster ID. This flag requires a project/domain ID, project/domain name will not work.").Short('c').String()
+
+	osAuthURL           = app.Flag("os-auth-url", "Authentication URL.").PlaceHolder("OS_AUTH_URL").String()
+	osUsername          = app.Flag("os-username", "Username").PlaceHolder("OS_USERNAME").String()
+	osUserDomainName    = app.Flag("os-user-domain-name", "User's domain name.").PlaceHolder("OS_USER_DOMAIN_NAME").String()
+	osProjectName       = app.Flag("os-project-name", "Project name to scope to.").PlaceHolder("OS_PROJECT_NAME").String()
+	osProjectDomainName = app.Flag("os-project-domain-name", "Domain name containing project to scope to.").PlaceHolder("OS_PROJECT_DOMAIN_NAME").String()
```

Comment:
The following variables are still missing: OS_PROJECT_ID/OS_DOMAIN_ID (for identifying a scope by its ID rather than its name) and OS_PASSWORD.